File: 04-Codigo-Clases/ProyectosJAVA/LP2SoftLogisticaModel/src/pe/edu/pucp/lp2soft/almacen/model/Producto.java
package pe.edu.pucp.lp2soft.almacen.model;
public class Producto {
    private int idProducto;
    private String nombre;
    private String unidadMedida;
    private double precio;
    private boolean activo;

    public Producto(String nombre, String unidadMedida, double precio) {
        this.nombre = nombre;
        this.unidadMedida = unidadMedida;
        this.precio = precio;
    }
    
    public int getIdProducto() {
        return idProducto;
    }

    public void setIdProducto(int idProducto) {
        this.idProducto = idProducto;
    }

    public String getNombre() {
        return nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    public String getUnidadMedida() {
        return unidadMedida;
    }

    public void setUnidadMedida(String unidadMedida) {
        this.unidadMedida = unidadMedida;
    }

    public double getPrecio() {
        return precio;
    }

    public void setPrecio(double precio) {
        this.precio = precio;
    }

    public boolean isActivo() {
        return activo;
    }

    public void setActivo(boolean activo) {
        this.activo = activo;
    }
    
    
}

File: 04-Codigo-Clases/ProyectosNET/LP2Soft/LP2SoftLogisticaModel/Almacen/Producto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace LP2SoftLogisticaModel.Almacen
{
    public class Producto
    {
        private int _idProducto;
        private string _nombre;
        private string _unidadMedida;
        private double _precio;
        private bool _activo;

        public int IdProducto { get => _idProducto; set => _idProducto = value; }
        public string Nombre { get => _nombre; set => _nombre = value; }
        public string UnidadMedida { get => _unidadMedida; set => _unidadMedida = value; }
        public double Precio { get => _precio; set => _precio = value; }
        public bool Activo { get => _activo; set => _activo = value; }
    }
}
